src/client/src/ui/analytics/epics/index.ts

```diff
@@ -1,11 +1,15 @@
 import { combineEpics } from 'redux-observable'
 import { publishPositionUpdateEpic } from './analyticsServiceEpic'
 import { analyticsServiceEpic } from './epics'
+import { InteropServices, PlatformAdapter } from 'rt-components'
 
-const epics = [analyticsServiceEpic]
+export default ({ platform }: { platform: PlatformAdapter }) => {
+  const { interopServices }: { interopServices: InteropServices } = platform
```

Comment:
You should not need to specify the type here. Just use ApplicationDependencies as Dan suggests.

Also, you do not always need to use a deconstructor, especially for nested properties.

`const interopServices = platform.interopServices`